packages/dds/tree/src/test/shared-tree/fuzz/fuzzUtils.ts

```diff
@@ -20,6 +21,7 @@ export function runFuzzBatch(
 		generatorFactory: AsyncGenerator<Operation, FuzzTestState>,
 		seed: number,
 		saveInfo?: SaveInfo,
+		savedInfo?: SavedInfo,
```

Comment:
I don't expect that we'll need to save a whole batch of fuzz runs and run them from file. We're much more likely to save whatever runs are failing and running them individually. Consequently, we don't need this function to be updated.